Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Use the existing ScienceBeam parser image as a base
+FROM elifesciences/sciencebeam-parser:latest
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the current directory contents into the container /app folder
+COPY . /app
+
+# Install dependencies in requirements.txt
+RUN pip install --upgrade pip \
+    && pip install -r requirements.txt
+
+# Expose port 80 outside this container
+EXPOSE 80
+
+# Run a sample command to validate set up
+CMD ["python", "osm/cli.py", "--help"]
```

Comment:
Setting entrypoint, cmd directives should ideally be done to conform to [best practices](https://docs.docker.com/build/building/best-practices/#entrypoint). 

I think we want to make things easy for our users so the osm cli should be the entrypoint. The default command can be the help docs.

I think this will probably work better as a docker swarm though. Otherwise you will have to define an entrypoint for the container that runs the sciencebeam parser as a service in the background.

I don't mind either way but the tests should take into account the deployment strategy. You may consider mocking the sciencebeam parser to ease the development of tests.

Reply:
Git it. We'll add an entry point and explore two options: using Docker Swarm or running the ScienceBeam Parser image as a background service. We'll decide based on deployment best practices.